scripts/controllers/Lobby.gd

```diff
@@ -21,15 +21,11 @@ remote func player_joined():
 	Net.current_players += 1
 	connected()
 
+
 remotesync func begin_game():
 	## TODO: Set the random seed here, and broadcast it to all players before the game starts
 	var _junk = get_tree().change_scene("res://scenes/World.tscn")
 
-func _on_Start_pressed():
-	rpc("begin_game")
-
 
-func _on_UpdateIPLabels_timeout():
-	# Update IPs
-	$Connecting/MyIP.text = "IP: " + str(Net.public_ip)
-	$Host/MyIP.text = "IP: " + str(Net.public_ip)
+func _on_BGMusic_finished():
+	pass #$BGMusic.play()
```

Comment:
No repeating music? :'(

Reply:
I know how much you love the sound of my smacking lips